spring-cloud-commons/src/main/java/org/springframework/cloud/client/DefaultServiceInstance.java

```diff
@@ -148,8 +180,8 @@ public void setUri(URI uri) {
 		this.uri = uri;
 		this.host = this.uri.getHost();
 		this.port = this.uri.getPort();
-		String scheme = this.uri.getScheme();
-		if ("https".equals(scheme)) {
+		scheme = this.uri.getScheme();
+		if (Arrays.asList("https", "wss").contains(scheme)) {
```

Comment:
Any concern on case?
I would make the list a class variable instead of creating it everytime setUri is called.